numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
currently there is only precompiled libraries. To get things work you should have the path of libraries in LD_ LIBRARY_PATH. Library files are in numerrin-interface directory. I think that all proprietary codes should be distributed as precompiled libraries to avoid licensing issues.